modules/admin-ui-picard/app/src/components/shared/modals/ResourceDetailsAccessPolicyTab.js

```diff
@@ -0,0 +1,457 @@
+                                                                                )}
+                                                                            </tr>
+                                                                        ))
+                                                                        }
+
+                                                                        {/* create additional policy */}
+                                                                        { !transactions.read_only /*Todo:   && $root.userIs('ROLE_UI_EVENTS_DETAILS_ACL_EDIT') */ && (
+                                                                            <tr>
+                                                                                <td colSpan="5">
+                                                                                    <a onClick={() => push(createPolicy(""))}>
+                                                                                        + {t("EVENTS.EVENTS.DETAILS.ACCESS.ACCESS_POLICY.NEW")}
+                                                                                    </a>
+                                                                                </td>
+                                                                            </tr>
+                                                                        )}
+                                                                    </>
+                                                                )}
+                                                            </FieldArray>
+                                                            </tbody>
+
+                                                        </table>
+                                                    </div>
+                                                </div>
+                                            </div>
+
+                                            {/* Save and cancel buttons */}
+                                            { (!transactions.read_only && policyChanged) && (
+                                                <footer style={{padding: '15px'}}>
+                                                    <div className="pull-left">
+                                                        <button type="reset"
+                                                                onClick={() => resetPolicies(formik.resetForm)}
+                                                                className="cancel"
+                                                        >
+                                                            {t('CANCEL')/* Cancel */}
+                                                        </button>
+                                                    </div>
+                                                    <div className="pull-right">
+                                                        <button onClick={() => saveAccess(formik.values)}
```

Comment:
Check here, if formik form is changed by the user and deactivate button if not. You can check this by using `formik.isValid ` and ` formik.dirty`.

Reply:
If formik form is changed is checked for whole footer. The whole footer with both buttons is only displayed, if the formik form is changed.
I added the _formik.isValid_. Now the button is displayed but disabled, if _!formik.isValid_.